addons/event/report/event_event_templates.xml

```diff
@@ -26,6 +26,8 @@
                                             <h3 t-field="o.name"/>
                                         </small>
                                     </div>
+                                     <div class="col-xs-12 text-center" style="background: lightgrey;" id="o_event_barcode">
```

Comment:
@mba-odoo Well in that case I will add two separate divs, in event_sale as well as in event_barcode:
``` <xpath expr="//div[@id='o_event_attendee_name']" position="after">
            <div class="col-xs-12 text-center" style="background: lightgrey;" id="o_event_barcode">
                <div t-if="bool(len(event.event_ticket_ids))" class="col-xs-12 mt16 text-center" style="background: lightgrey; height: 65px;" t-ignore="true">
                    <h3>Ticket Type</h3>
                </div>
            </div>
        </xpath> ```

Same way in event barcode, but adding void div will add unnecessary grey line